src/components/templates/Page.tsx

```diff
@@ -27,7 +27,7 @@ export default function Page({
           title={title}
           description={description}
           center={headerCenter}
-          powered
+          powered={uri === '/'}
```

Comment:
suggestion(non-blocking): extracting to a variable like `isHome` might improve readability